classes/projects-define.php

```diff
@@ -59,6 +59,9 @@ public function define_post_type() {
 					'api_key'          => new \Fieldmanager_TextField( array(
 						'label'          => __( 'API Key', 'project-dashboard' ),
 						'serialize_data' => false,
+						'attributes' => array( 'readonly' => true, 'disabled' => true    ),
+						'default_value' => wp_generate_password( 12 ),
```

Comment:
Seems like it would overly complicate the requests. The request will be done via shell scripts on a CI tool, and would require it to be escaped prior to send over, right?